test/functional/test_ipmi_console.py

```diff
@@ -245,8 +245,8 @@ def test_ipmi_console_stops_followed_by_node_destroy(self):
                 stderr=subprocess.PIPE)[1]
             if start_ipmi_console_cmd not in output:
                 break
-            else:
-                assert False
+        else:
+            assert False
```

Comment:
Wrong indent of above two lines leads to test failure.
Be careful about IDE petty trick 🐶

Reply:
🐩